docs/source/examples/Notebook/rstversions/Typesetting Equations.rst

```diff
@@ -50,7 +50,7 @@ Source
     \end{equation*}
 
 Display
--------
+~~~~~~~
 
```

Comment:
@takluyver I just saw your note here https://github.com/jupyter/notebook/commit/884ef1dc28329741f62978cbd7c1fa5d3c00bde9 before the note on this issue. I think this rst doc is being autogenerated by pandoc. Perhaps a post pandoc processing script to search and replace the ``.. raw::latex`` with ``.. math::``. Hack but might do the trick.